### DISCUSSION TITLE
Auto updater

### DISCUSSION THREAD

**Discussion by binaryfire** [NativePHP/laravel#404] [Ideas]
Hi

Just wondering if it's possible to add an auto-updater? Most Electron apps show a "New version available" popup with a button to do a one-click update. I'm guessing this would need to be handled outside of the PHP layer? And perhaps poll a Github releases url.

**Comment by simonhamp** [comment]
We already have an [updater](https://nativephp.com/docs/1/publishing/updating)

**Reply by binaryfire** [reply]
@simonhamp No idea how I missed that when reading the docs… Nice, thanks.
